com.unity.ml-agents/Runtime/SideChannels/TrainingAnalyticsSideChannel.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using UnityEngine;
+namespace Unity.MLAgents.SideChannels
+{
+    public class TrainingAnalyticsSideChannel : SideChannel
+    {
+        const string k_TrainingAnalyticsConfigId = "b664a4a9-d86f-5a5f-95cb-e8353a7e8356";
+
+        /// <summary>
+        /// Initializes the side channel. The constructor is internal because only one instance is
+        /// supported at a time, and is created by the Academy.
+        /// </summary>
+        internal TrainingAnalyticsSideChannel()
+        {
+            ChannelId = new Guid(k_TrainingAnalyticsConfigId);
+        }
+
+        /// <inheritdoc/>
+        protected override void OnMessageReceived(IncomingMessage msg)
+        {
+            var eventName = msg.ReadString();
+            if (!string.IsNullOrEmpty(eventName))
+            {
+                Debug.Log(eventName);
```

Comment:
Confirmed that this gets 
```
environment_initialized!
```
and
```
training_started for 3DBall with config TrainerSettings(trainer_type=<TrainerType.PPO: 'ppo'>, ...)!
```
when using 3DBall. This is obviously placeholder; the plan is to add a protobuf message and send that over the SideChannel instead.